addons/l10n_ar/models/account_journal.py

```diff
@@ -9,14 +9,27 @@ class AccountJournal(models.Model):
     _inherit = "account.journal"
 
     l10n_ar_afip_pos_system = fields.Selection(
-        selection='_get_l10n_ar_afip_pos_types_selection', string='AFIP POS System')
+        selection='_get_l10n_ar_afip_pos_types_selection', string='AFIP POS System',
+        compute='_compute_l10n_ar_afip_pos_system', store=True, readonly=False)
     l10n_ar_afip_pos_number = fields.Integer(
         'AFIP POS Number', help='This is the point of sale number assigned by AFIP in order to generate invoices')
     company_partner = fields.Many2one('res.partner', related='company_id.partner_id')
     l10n_ar_afip_pos_partner_id = fields.Many2one(
         'res.partner', 'AFIP POS Address', help='This is the address used for invoice reports of this POS',
         domain="['|', ('id', '=', company_partner), '&', ('id', 'child_of', company_partner), ('type', '!=', 'contact')]"
     )
+    l10n_ar_is_pos = fields.Boolean(
+        compute="_compute_l10n_ar_is_pos", store=True, readonly=False, string="Is AFIP POS?")
+
+    @api.depends('country_code', 'type', 'l10n_latam_use_documents')
+    def _compute_l10n_ar_is_pos(self):
+        for journal in self:
+            journal.l10n_ar_is_pos = journal.country_code == 'AR' and journal.type == 'sale' and journal.l10n_latam_use_documents
+
+    @api.depends('l10n_ar_is_pos')
+    def _compute_l10n_ar_afip_pos_system(self):
+        for journal in self:
+            journal.l10n_ar_afip_pos_system = journal.l10n_ar_is_pos and journal.l10n_ar_afip_pos_system
```

Comment:
This is a weird compute.  Why not an onchange to set values back?  Or is this the new style?